Tensile/KernelWriterAssembly.py

```diff
@@ -8071,7 +8072,8 @@ def computeStoreSrdStart(self, kernel):
         strideC = "StrideC%s"%self.indexChars[i]
         kStr += self.s_mul_u64_u32(sgpr(tmpS0), sgpr(tmpS1), coord, sgpr(strideC), "CScale %s by Stride"%coord)
         #kStr += assert_no_shift_of(tmpS1, log2(self.bpeCexternal), "Need temp")
-        kStr += inst("s_lshl_b64", sgpr(tmpS0,2), sgpr(tmpS0,2), log2(self.bpeCexternal), "scale by bpe")
+        tmpBpe = self.bpr if kernel["_GlobalAccumulation"] else self.bpeCexternal
```

Comment:
can we use `bpeCinternal` here to denote the size requirement for accumulation?

Reply:
update. use bpeCinternal to replace self.bpr